### PR TITLE
DATA-26 - Add user role to completion report.

### PR DESCRIPTION
**Pull request by Squirrel18**
## Description:

This PR adds the user role for completion report.

## Reviewers:

- [ ] @andrey-canon

### CODE REVIEW

File: openedx_proversity_reports/reports/last_page_accessed.py [hunk 1]
```diff
@@ -57,13 +58,9 @@ def get_last_page_accessed_data(course_list):
...
             parent_tree_name = ''
             vertical_block_id = ''
-            user_role = 'student'
 
             if last_completed_child_position:
-                user_course_role = course_access_role().objects.filter(
-                    user=user,
-                    course_id=course_key
-                )
+                user_course_role = get_user_role(user, course_key)
```
Comment (andrey-canon) [inline openedx_proversity_reports/reports/last_page_accessed.py]: This line is not necessary , you can call the method directly in the line 80

File: openedx_proversity_reports/utils.py [hunk 2]
```diff
@@ -53,23 +54,18 @@ def update_user_dict(child, user_data, section=None, subsection=None, vertical=N
     data = []
     for user in users:
         block_data = copy.deepcopy(root_block)
-        if user_has_role(user, get_course_staff_role(course_key)):
-            continue
         mark_blocks_completed(block_data, user, course_key)
         sections = block_data.get('children', [])
+        user_role = get_user_role(user, course_key)
```
Comment (andrey-canon) [inline openedx_proversity_reports/utils.py]: same comment as above

File: openedx_proversity_reports/utils.py [hunk 3]
```diff
@@ -220,3 +216,44 @@ def recurse_mark_complete(course_block_completions, latest_completion, block):
...
+        courseaccessrole__role='staff',
+    ).first()
+
+    return staff_user
+
+
+def get_user_role(user, course_key):
+    """
+    Returns the user string role.
+    The default role it's student.
```
Comment (andrey-canon) [inline openedx_proversity_reports/utils.py]: Grammatical error ?? I don't understand

File: openedx_proversity_reports/reports/last_page_accessed.py (hunk 1 above)
Comment (Squirrel18) [reply openedx_proversity_reports/reports/last_page_accessed.py]: Done.

File: openedx_proversity_reports/utils.py (hunk 2 above)
Comment (Squirrel18) [reply openedx_proversity_reports/utils.py]: Done.

File: openedx_proversity_reports/utils.py (hunk 3 above)
Comment (Squirrel18) [reply openedx_proversity_reports/utils.py]: Changed.

**Comment by Squirrel18** [comment]
Check again. @andrey-canon

**Pull request merged**